dashboard/app/models/lesson.rb

```diff
@@ -494,6 +494,10 @@ def update_from_curriculum_builder(cb_lesson_data = {})
       self.creative_commons_license = cb_lesson_data['creative_commons_license']
       self.relative_position = cb_lesson_data['number'] || 1
       self.lesson_activities = Services::LessonImportHelper.create_lesson_activities(cb_lesson_data['activities'], script_levels, id)
+      course_version_id = script&.get_course_version&.id
+      # course version id should always be present for CSF/CSD/CSP 2020 courses.
+      raise unless course_version_id
```

Comment:
Is it worth doing this check before creating the lesson activities? Otherwise we'll end up with lessons with new lesson activities, no resource, with script levels still assigned to the lesson. Of course, these lessons are already messed up in this case so I'm not strongly opinionated either way.